code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -484,14 +484,20 @@
 
 	. = 0
 
+///Determines the rate at which Plasma Fixation heals based on the amount of plasma in the air
+
+#define HEALING_PER_MOL 1.1
```

Comment:
```suggestion
///Determines the rate at which Plasma Fixation heals based on the amount of plasma in the air
#define HEALING_PER_MOL 1.1
```
No newline line between the comment and the define!

Reply:
AHHHHHHHHHHHHHHHH

Reply:
...thank you for taking the time to review this 😭